tests/Plaid.Tests/Tests/PlaidClientTest.cs

```diff
@@ -52,6 +52,7 @@ public async Task InitializeAsync()
 						Products.Balance,
 						Products.Investments,
 						Products.Transactions,
+						Products.Assets
```

Comment:
Always use a comma on the last item of a list in C#, to reduce future code changes when adding another item to the list.

```suggestion
						Products.Assets,
```

Reply:
Resolve button hates me. But yes, this is updated.